World/AbstractWorld.h

```diff
@@ -50,10 +50,4 @@ class AbstractWorld {
               << std::endl;
```

Comment:
evaluate should be made pure virtual, so the compiler can provide this error.

Reply:
Yup. change made.